langlib/langlib-test/src/test/resources/test-src/tablelib_test.bal

```diff
@@ -42,6 +61,14 @@ type CustomerTable table<Customer> key(id);
 
 type CustomerKeyLessTable table<Customer>;
 
+type PersonTable table<Person>;
+
+type EngineerTable table<Engineer>;
+
+type InternTable table<Intern>;
+
+type PersonAnyTable table<map<any>>;
```

Comment:
Shall we do use a map<string> as table constrained and try to add () as well

Reply:
It's already there. 
https://github.com/ballerina-platform/ballerina-lang/blob/b223b7551f1aa5f3c532a6457e3037f4b4722c6b/langlib/langlib-test/src/test/resources/test-src/tablelib_test.bal#L367